VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitWebviewFragment.java

```diff
@@ -111,8 +107,23 @@ public void onReceivedError(WebView view, int errorCode,
                 hideLoadingProgress();
                 pageIsLoaded = false;
                 ViewPagerDownloadManager.instance.done(CourseUnitWebviewFragment.this, false);
-                super.onReceivedError(view, errorCode, description, failingUrl);
             }
+
+            // Restore these annotations when we upgrade our compile SDK version to Marshmallow
```

Comment:
The annotations are not necessary for the code to work in Marshmallow. The override annotation simply ensures that the compiler will know that this method is supposed to override a base method, and report an error if it does not match the signature of a method in a superclass. The `Build.VERSION_CODES.M` constant should probably be used instead of defining the SDK version literally, but it is not yet available in our compile SDK version (Lollipop), and the constant name will probably be changed to `Marshmallow` in any case.